onnxruntime/core/graph/contrib_ops/quantization_defs.cc

```diff
@@ -975,51 +993,61 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
-          .SetDoc(R"DOC(Quantize input matrix to specific layout used in cublaslt.)DOC")
-          .Attr("order_input",
-                "cublasLt order of input matrix. ORDER_COL = 0, ORDER_ROW = 1, ORDER_COL32 = 2, ORDER_COL4_4R2_8C = 3, ORDER_COL32_2R_4R4 = 4. "
-                "Please refer https://docs.nvidia.com/cuda/cublas/index.html#cublasLtOrder_t for their meaning.",
-                AttributeProto::INT)
-          .Attr("order_output", "cublasLt order of output matrix.", AttributeProto::INT)
-          .Input(0, "input", "TODO: input tensor of (ROWS, COLS). if less than 2d, will broadcast to (1, X). If 3d, it is treated as (B, ROWS, COS)", "F")
-          .Input(1, "scale_input", "scale of the input", "S")
-          .Output(0, "output", "output tensor", "Q")
-          .TypeConstraint("Q", {"tensor(int8)"}, "Constrain input and output types to int8 tensors.")
-          .TypeConstraint("F", {"tensor(float16)", "tensor(float)"}, "Constrain to float types")
-          .TypeConstraint("S", {"tensor(float)"}, "Constrain Scale to float32 types")
-          .TypeAndShapeInferenceFunction([](ONNX_NAMESPACE::InferenceContext& ctx) {
-            propagateElemTypeFromDtypeToOutput(ctx, ONNX_NAMESPACE::TensorProto::INT8, 0);
-            if (!hasInputShape(ctx, 0)) return;
-            auto& input_shape = getInputShape(ctx, 0);
-            updateOutputShape(ctx, 0, input_shape);
-          }));
-
-  ONNX_MS_OPERATOR_SET_SCHEMA(
-      DequantizeWithOrder,
-      1,
-      OpSchema()
-          .SetDoc(R"DOC(Dequantize input matrix to specific layout used in cublaslt. attr to specify output type, float16 or float32)DOC")
-          .Attr("order_input", "cublasLt order of input matrix. See the schema of QuantizeWithOrder for order definition.", AttributeProto::INT)
-          .Attr("order_output", "cublasLt order of output matrix", AttributeProto::INT)
-          .Attr("to", "The output data type, only support TensorProto_DataType_FLOAT (1) and TensorProto_DataType_FLOAT16 (10)", AttributeProto::INT)
-          .Input(0, "input", "TODO: input tensor of (ROWS, COLS). if less than 2d, will broadcast to (1, X). If 3d, it is treated as (B, ROWS, COS)", "Q")
-          .Input(1, "scale_input", "scale of the input", "S")
-          .Output(0, "output", "output tensor", "F")
-          .TypeConstraint("Q", {"tensor(int8)"}, "Constrain input and output types to int8 tensors.")
-          .TypeConstraint("F", {"tensor(float16)", "tensor(float)"}, "Constrain to float types")
-          .TypeConstraint("S", {"tensor(float)"}, "Constrain Scale to float32 types")
-          .TypeAndShapeInferenceFunction([](ONNX_NAMESPACE::InferenceContext& ctx) {
-            propagateElemTypeFromAttributeToOutput(ctx, "to", 0);
-            if (!hasInputShape(ctx, 0)) return;
-            auto& input_shape = getInputShape(ctx, 0);
-            updateOutputShape(ctx, 0, input_shape);
-          }));
+ONNX_MS_OPERATOR_SET_SCHEMA(
+    QuantizeWithOrder, 1,
```

Comment:
what is this schema?

Reply:
These changes are formatting only and are a result of running shift+alt+f (clang-format) on the source file.